TSOClient/tso.simantics/NetPlay/Model/Commands/VMBudgetCheatCmd.cs

```diff
@@ -0,0 +1,34 @@
+﻿using FSO.SimAntics.Utils;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FSO.SimAntics.NetPlay.Model.Commands
+{
+    /// <summary>
+    /// A command submitting a cheat with a typed parameter T
+    /// </summary>
+    public class VMBudgetCheatCmd : VMNetCommandBodyAbstract
```

Comment:
This has no serialization so cannot be networked, but I suppose it's not a concern right now.

Reply:
While I'm suggesting the other change, this should just be VMNetCheatCmd, as it covers more than budget.